software/externals/cmake/externals.cmake

```diff
@@ -307,6 +307,29 @@ if(NOT APPLE)
 endif()
 
 
+if(USE_SYSTEM_OPENCV)
+  list(REMOVE_ITEM externals
+    opencv
+  )
+  list(REMOVE_ITEM occ-map_depends
+    opencv
+  )
+  list(REMOVE_ITEM libmultisense_depends
+    opencv
+  )
+endif()
```

Comment:
I've seen you use ${lcm_proj} which I reckon is empty if not defined - would that be a cleaner way for this? (Happy to amend)